test/integ/cli/job_templates.py

```diff
@@ -0,0 +1,137 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Job template utilities for incremental download tests.
+"""
+
+from pathlib import Path
+from typing import Optional
+
+from deadline.client.api._submit_job_bundle import create_job_from_job_bundle
+from deadline.client.config import config_file
+
+
+def get_job_bundle_path(template_name: str) -> str:
+    """
+    Get the path to a job bundle template directory.
+
+    Args:
+        template_name: Name of the template (e.g., 'make_many_small_files')
+
+    Returns:
+        Path to the template directory
+    """
+    current_dir = Path(__file__).parent
+    bundle_path = current_dir / "job_bundles" / template_name
+
+    if not bundle_path.exists():
+        raise FileNotFoundError(f"Job bundle template not found: {bundle_path}")
+
+    return str(bundle_path)
+
+
+def submit_job_bundle(
+    farm_id: str, queue_id: str, template_name: str, parameters: Optional[dict] = None
+) -> str:
+    """
+    Submit a job using a local job bundle template.
+
+    Args:
+        farm_id: The farm ID to use
+        queue_id: The queue ID to use
+        template_name: Name of the template directory
+        parameters: Optional parameters to pass to the job
+
+    Returns:
+        The job ID of the submitted job
+    """
+    bundle_path = get_job_bundle_path(template_name)
+
+    # Convert parameters to the format expected by create_job_from_job_bundle
+    job_parameters = []
+    if parameters:
+        job_parameters = [{"name": key, "value": value} for key, value in parameters.items()]
+
+    # Set farm and queue in config
+    config = config_file.read_config()
+    config.set("defaults", "farm_id", farm_id)
+    config.set("defaults", "queue_id", queue_id)
```

Comment:
When I ran these tests, I found they submitted to the wrong queue. With this change, they went to the right one:

```suggestion
    config_file.set_setting("defaults.farm_id", farm_id, config)
    config_file.set_setting("defaults.queue_id", queue_id, config)
```

Reply:
Chatted more about this offline. Looks like this happens when the defaults conflict with the env variable. Inside deadline config, ``queue_id`` is nested under ``farm_id``. Since this change set the top level queue_id, that gets ignored, and it continues using the default queue id. Didn't happen on my dev testing because the ``queue_id`` being used in the env variables was the same as the default queue in the deadline config so there was no conflict. Fixed it